DuggaSys/sectioned.php

```diff
@@ -186,8 +186,8 @@ function updateGithubDir($pdo, $githubDir, $lid) {
 				<img src="../Shared/icons/desc_complement.svg" alt='Hide List Content' id="sectionList_arrowStatisticsClosed">
 			</div>
 			<div class='fixed-action-button2 sectioned2'  id="FABStatic2" style="display:none">
-				<input id='addElement'  type='button' value='+' style="margin-right:20px; top:-493px" class='submit-button-newitem' title='New Item' >
-				<ol class='fab-btn-list2' style='display: none;'  reversed id='fabBtnList2'>
+				<input id='addElement'  type='button' value='+' style="margin-right:20px; top:-493px; position:fixed;" class='submit-button-newitem' title='New Item' >
```

Comment:
Fix spacing between attributes, id - type has 2 spaces for instance... Maybe not within the scope of your issue, but would prefer inline-css to be removed since input has an id, and could easily be moved to the css-file